app/packages/core/src/components/Common/Input.tsx

```diff
@@ -74,6 +74,7 @@ const Input = React.memo(
           <StyledInput
             ref={ref}
             placeholder={placeholder}
+            data-cy={`baseinput`}
```

Comment:
hm... this is not a greta data-cy name I'd say. Can you be more descriptive? no {} required also